qutebrowser/extensions/loader.py

```diff
@@ -118,6 +121,42 @@ def _walk_normal() -> typing.Iterator[ExtensionInfo]:
         yield ExtensionInfo(name=name)
 
 
+def walk_extensions() -> typing.Iterator[ExtensionInfo]:
+    """Walk external extensions."""
+    ext_dir = os.path.join(standarddir.data(), 'extensions')
+    if not os.path.exists(ext_dir):
+        try:
+            os.mkdir(ext_dir)
+        except OSError:
+            pass
+        return
+
+    if any(os.listdir(ext_dir)):
```

Comment:
`any()` takes a list of values and checks whether any of those is boolean-True. It happens to work here because `any([]) == False` and `any(['foo', 'bar']) == any([True, True]) == True`, but just

```suggestion
    if os.listdir(ext_dir):
```

(relying on the list to be either empty or not) is far less confusing :wink: